src/lib.rs

```diff
@@ -320,8 +320,18 @@ impl Builder {
 		self
 	}
 
+	/// Builds a [`Node`] instance with a [`FilesystemStore`] backend and according to the options
+	/// previously configured.
+	pub fn build(&self) -> Arc<Node<FilesystemStore>> {
+		let ldk_data_dir = format!("{}/ldk", self.config.storage_dir_path);
+		let kv_store = Arc::new(FilesystemStore::new(ldk_data_dir.clone().into()));
+		self.build_with_store(kv_store)
+	}
+
 	/// Builds a [`Node`] instance according to the options previously configured.
-	pub fn build(&self) -> Arc<Node> {
+	pub fn build_with_store<K: KVStore + Sync + Send + 'static>(
```

Comment:
can't we do similar to set_esplora_server_url ?
and use similar builder pattern for this.

Reply:
there's one more related or unrelated item. 
some KVStore's like Vss might need a runtime, 

Current process of build_with_store assumes kvstore will be provided externally, but we will have no way to inject common runtime without something like set_runtime.
any thoughts or solutions to that?